state_postal_code_to_fips.json

```diff
@@ -0,0 +1,58 @@
+{
+  "AL": "1",
```

Comment:
I do wonder if there will be some cases where we need e.g. "01" instead of "1" but I know that pandas by default reads in the CSV numbers as "1" instead of "01", so this is what we need for a join.